GrammarLinter.md

```diff
@@ -0,0 +1,81 @@
+## English Grammar Linter (Vale)
+
+This repository includes `beta` rules based on the [Vale grammar linter](https://docs.errata.ai/vale). To install Vale on a Mac, run:
```

Comment:
The link in this sentence seems to be a dead link, I'm getting a Page Not Found error. I'm not sure what you intend to link to, maybe `https://docs.errata.ai/#vale` (with `#`)?

Reply:
```suggestion
This repository includes `beta` rules based on the [Vale grammar linter](https://docs.errata.ai/#vale). To install Vale on a Mac, run:
```